testutil/runbook.go

```diff
@@ -0,0 +1,71 @@
+package testutil
+
+import (
+	"fmt"
+	"html/template"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+// BenchmarkSet returns a server and a path pattern for benchmarking.
+func BenchmarkSet(t testing.TB, bookCount, stepCount int, body string) (*httptest.Server, string) {
+	const httpRunbookTmpl = `desc: Test using HTTP
+runners:
+  req: {{ .Endpoint }}
+steps:
+{{ range $k := .Steps }}
+-
```

Comment:
I want to set up a label and debug setting.

Reply:
Why do you want to enable `label` or `debug`?